src/services/Authentication/getLoginResponse.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+const getLoginResponse = async ({ email, password }) => {
+    try {
+        const loginResponse = await axios.post("/api/auth/login", {
```

Comment:
Can destructure data here only

Reply:
Have destructured it while writing the submission handler function for Login and Signup operations. Here I intend to collect the whole response whenever Signup and Login happens.